### PR TITLE
basic hello world package - taken from rumprun project

### PR DESCRIPTION
**Pull request by useidel**
For an even easier start I have created a package which basically re-uses the "Hello World" example from the rumprun project and creates the package for Nabla/Rumprun/Solo5 framework.

### CODE REVIEW

**Comment by useidel** [comment]
There is a dependency on a pull request of a related repository:  http://github.com/nabla-containers/nabla-base-build/pull/10

File: hello/Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+include ../Makefile.inc
+UPSTREAM=hello.tar.gz
```
Comment (ricarkol) [inline hello/Makefile]: Would prefer to have the source files uncompressed, so people can play with them.

**Review by ricarkol — CHANGES_REQUESTED** [review]
All good, thanks! just one nit about the compressed source files.

File: hello/Makefile (hunk 1 above)
Comment (useidel) [reply hello/Makefile]: Ah - ok. Okies ... will do so.

**Comment by useidel** [comment]
code updated as advised

**Comment by ricarkol** [comment]
Thanks

**Pull request merged**